src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const element = document.body.children[1];
+const list = element.children;
+
+const sumString = string => +string.slice(1).split(',').join('');
+
+const sortList = function() {
+  return [...list].sort((a, b) =>
+    sumString(b.dataset.salary) - sumString(a.dataset.salary));
+};
+
+element.append(...sortList(list));
```

Comment:
```suggestion
element.append(...sortList());
```
pay attention that now since `sortList` function doesn't have any parameters, you can remove `list` argument. Or back the parameter to `sortList` and use it inside function instead of `list`